packages/core/src/runtime/ast/executableExpression.ts

```diff
@@ -1,13 +1,14 @@
-import { Expression } from "../../ast/expression";
-import { InterpreterContext } from "../interpreter";
-import { FieldEntry } from "../objects/baseObject";
+import { Expression } from "../../ast/expression.js";
+import { InterpreterContext } from "../interpreter.js";
+import { FieldEntry } from "../objects/baseObject.js";
+// DO NOT CHANGE! This is required to make circular imports work
```

Comment:
WHAT shouldn't be changed?

Reply:
261fcf3c43a170f79d44aadc4c7b5e8e0d5dd993
clearer?

Reply:
Yes.
The only remaining question is now: The ones above or below this line?
However, since there are only two options, it should be quite easy to discover which ones you are talking about